index.html

```diff
@@ -2,44 +2,36 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="A web playground to write and share FHIR Shorthand. It's so fun, you're guaranteed to get hooked!" />
 
     <meta property="og:title" content="FSH Online"/>
     <meta property="og:description" content="A web playground to write and share FHIR Shorthand. It's so fun, you're guaranteed to get hooked!"/>
     <meta property="og:url" content="/FSHOnline"/>
-    <meta property="og:image" content="%PUBLIC_URL%/FSH-online-logo.png"/>
+    <meta property="og:image" content="/FSH-online-logo.png"/>
 
     <meta name="twitter:title" content="FSH Online"/>
     <meta name="twitter:description" content="A web playground to write and share FHIR Shorthand. It's so fun, you're guaranteed to get hooked!"/>
     <meta property="twitter:url" content="/FSHOnline"/>
     <meta name="twitter:card" content="summary"/>
-    <meta name="twitter:image" content="%PUBLIC_URL%/FSH-online-logo.png"/>
+    <meta name="twitter:image" content="/FSH-online-logo.png"/>
 
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="apple-touch-icon" href="/favicon.ico" />
```

Comment:
Was this just wrong before?

Reply:
Yeah it was. You can see this even on the currently deployed version. There's a 404 error because it can't find that file. We deleted that file, which was a React logo added by CRA, but we never updated the reference to it. I believe the `apple-touch-icon` is used as the app cover, so I don't think it ever affected anything on the browser, so it hadn't really mattered, but I just cleaned it up while I was in the file.